<div class="research-banner-bg">
  <div class="container">
     <div class="row">
       <div class="col-md-12" align="right">
         <h3><font color="grey">Expanding the horizon of computational biology research</font></h3>
       </div>
     </div>
  </div>
</div>

<!-- RESEARCH -->
<div class="container-fluid">
  <div class="row">
    <div class="col-md-12" align="left">
      <!-- <h2>Research</h2> -->
      <hr />
      <h4>The Computational Molecular Biology group is pursuing 4 major research topics as well as collaborative projects 
      in diverse biomedical research areas. Most projects combine broad knowledge of molecular biology with state-of-the-art mathematical 
      modeling and machine learning techniques.</h4>
    </div>
  </div>
  
  <div class="row">
  	<div class="col-md-12">
  	  <hr />
      <h3><strong>Project 1:</strong> Proteomics data analysis method and software development</h3>
    </div>
    <div class="col-md-12">
         <p>The goal of this project is to develop high-performance, high-throughput analysis methods and software tools for proteomics datasets. 
         We are particularly interested in applying machine learning techniques on both CPU and GPU architectures to improve peptide identification 
         and quantification from mass spectrometry data. Example application areas include identification of (i) crosslinked peptides whose mass 
         spectra are generally 3-4 times more complex than those found in typical proteomics studies and (ii) novel peptide sequences and 
         modifications that currently do not exist in reference databases.</p>
       </div>
     </div>
     
     <div class="row">
    <div class="col-md-12" align="left">
      <hr />
      <h3><strong>Project 2:</strong> Integrative analysis of high-throughput biomedical datasets</h3>
    </div>
  </div>
  <div class="row">
       <div class="col-md-12">
         <p>For this project, our goal is to identify biological bases behind diseases or other complex biological systems through computational analyses of 
         high-throughput biomedical datasets. This can be either hypothesis-driven, where publicly available datasets are used to support/disprove a hypothesis, 
         or data-driven, where emergence of new datasets or technology dictates research direction. Therefore, topics of interest will usually be chosen based 
         on the availability of public databases or via collaborations with experimental researchers from inside and outside of the university.</p>
       </div>
     </div>
  
  <div class="row">
    <div class="col-md-12" align="left">
      <hr />
      <h3><strong>Project 3:</strong> Mathematical modeling and simulation of chromatin folding process</h3>
    </div>
  </div>
  <div class="row">
       <div class="col-md-12">
         <p>Genomes are usually thought of as 1-dimensional sequences of genetic codes. However, they are located on chromosomes 
         which fold in 3-dimensional space to form loops, topologically associated domains, and other higher-order structures. These 
         structures can have large impacts on gene expression regulation and evolution. For example, genes located in the same 
         domain may be co-regulated by sharing the same transcription factor molecules. Chromosome packing can also limit the 
         accessibility of genes in certain regions to transcription machineries and consequently control their activity.</p>
         
         <p>In this project, theoretical behaviors of popular models will be examined in conjunction with chromosome conformation 
         capture datasets from various organisms to gain further insight into the folding process of chromosomes.
       </div>
     </div>
  <div class="row">
    <div class="col-md-12" align="left">
      <hr />
      <h3><strong>Project 4:</strong> Curation and interpretation of research trends and other statistics in scientific literatures</h3>
    </div>
  </div>
  <div class="row">
    <div class="col-md-12">
      <p>To conduct effective research, one must rigorously study the literatures and choose the appropriate experimental protocols and data analysis tools.
      However, new research reports and academic software get published at ever-increasing rates that can overwhelm even the most seasoned researchers.
      For example, given the sheer number of <a href="https://en.wikipedia.org/wiki/List_of_RNA-Seq_bioinformatics_tools" target="_blank">software tools</a> 
      for processing RNA-seq data, it is not possible to select the most appropriate tools without examining the original articles for detailed descriptions 
      and other articles that employed the tools for usage examples. Furthermore, some popular, highly cited tools may have since been replaced by 
      emerging, better-performing alternatives. In this project, our first goal is to develop a curator that can recommends academic software tools 
      that suit a user's need. More complicated applications of such curation system will be explored over time.</p>
    </div>
  </div>
  <!-- 
  <div class="row">
    <div class="col-md-12" align="left">
      <hr />
      <h3><strong>Project 4:</strong> Visualization of biomedical data using virtual reality</h3>
    </div>
  </div>
  <div class="row">
    <div class="col-md-12">
      <p>Even though we can perceive three dimensions, our visualization of data has largely been limited to 2-dimension media such as monitor screen or a 
      piece of paper. With the emergence of affordable virtual reality technology, we can now explore the possibility of representing structural and 
      non-structural biomedical data in 3-dimensional space and subsequent visualization in virtual reality. A primary goal of this project is to develop 
      novel visualization methods and tools that facilitate comprehensive integration, visualization, and interpretation of complex biomedical datasets.</p>
    </div>
  </div>
  -->
  
</div>